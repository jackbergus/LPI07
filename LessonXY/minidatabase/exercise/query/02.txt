load table/group.txt
group C by B with + AS C
print