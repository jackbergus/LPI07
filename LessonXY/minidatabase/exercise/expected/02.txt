B,C
2,7
5,20
8,9
11,23
